src/styles/themes/default.js

```diff
@@ -86,11 +85,11 @@ const darkTheme = {
 
 darkTheme.PAGE_BACKGROUND_COLORS = {
     [SCREENS.HOME]: darkTheme.sidebar,
+    [SCREENS.SAVE_THE_WORLD.ROOT]: colors.tangerine800,
     [SCREENS.SETTINGS.PREFERENCES]: colors.blue500,
     [SCREENS.SETTINGS.WORKSPACES]: colors.pink800,
-    [ROUTES.SETTINGS_STATUS]: colors.green700,
-    [ROUTES.I_KNOW_A_TEACHER]: colors.tangerine800,
-    [ROUTES.SETTINGS_SECURITY]: colors.ice500,
+    [SCREENS.SETTINGS.SECURITY]: colors.ice500,
+    [SCREENS.SETTINGS.STATUS]: colors.green700,
 };
```

Comment:
Let's add a comment for why these need to be screen names

Reply:
Done

Reply:
I could see how this was confusing. The actual animation happens [here](https://github.com/Expensify/App/blob/d55d16699447464305f26ef834db79d752cef601/src/libs/Navigation/NavigationRoot.js#L102) and you might see that and think "currentRoute.name, oh forsure I need the route from ROUTES.ts", but no – it's actually the screen name. Like I explained in the comment I added to the code `ROUTEs.ts` is basically just URLs for web and for deep-linking.